.github/workflows/release-artifacts.yaml

```
@@ -50,18 +50,21 @@ jobs:
         podman cp kantra-download:/usr/local/static-report . && zip -r kantra.linux.${{ matrix.arch }}.zip static-report  
         podman cp kantra-download:/opt/rulesets . && zip -r kantra.linux.${{ matrix.arch }}.zip rulesets
         podman cp kantra-download:/usr/local/etc/maven.default.index . && zip -r kantra.linux.${{ matrix.arch }}.zip maven.default.index
+        podman cp kantra-download:/usr/local/etc/task.gradle . && zip -r kantra.linux.${{ matrix.arch }}.zip task.gradle
               
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)